openmc/mgxs/library.py

```diff
@@ -245,7 +247,8 @@ def by_nuclide(self, by_nuclide):
 
     @domain_type.setter
     def domain_type(self, domain_type):
-        cv.check_value('domain type', domain_type, tuple(openmc.mgxs.DOMAIN_TYPES))
+        cv.check_value('domain type', domain_type,
+                       tuple(openmc.mgxs.DOMAIN_TYPES))
```

Comment:
Oh yeah, if you're just checking values a list is fine. It's only an issue for `check_type` because `isinstance(...)` expects a type or tuple of types, never a list. Since this is checking values, I take back my first comment and replace it with "no need to cast to tuple".